webrender/res/ellipse.glsl

```diff
@@ -84,9 +84,11 @@ float clip_against_ellipse_if_needed(
       return current_distance;
     }
 
-    return distance_to_ellipse(pos - ellipse_center_radius.xy,
-                               ellipse_center_radius.zw,
-                               aa_range);
+    float distance = distance_to_ellipse(pos - ellipse_center_radius.xy,
+                                         ellipse_center_radius.zw,
+                                         aa_range);
+
+    return max(distance, current_distance);
```

Comment:
the semantics of `current_distance` is not clear to me. Shouldn't this be `min(distance, current_distance)`?

Reply:
We want to take the maximum distance, so that if any of the active clip regions results in clipping the primitive out, it removes it.